build-logic/publication/src/main/kotlin/convention.publication-base.gradle.kts

```diff
@@ -31,6 +31,26 @@ publishing.publications.withType<MavenPublication> {
                 name.set("Eugene Matsyuk")
                 url.set("https://github.com/matzuk")
             }
+            developer {
+                id.set("AzamatCherchesov")
+                name.set("Azamat Cherchesov")
+                url.set("https://github.com/AzamatCherchesov")
+            }
+            developer {
+                id.set("Nikitae57")
+                name.set("Nikita Evdokimov")
+                url.set("https://github.com/Nikitae57")
+            }
+            developer {
+                id.set("Fullik")
+                name.set("Pogos Azaryan")
+                url.set("https://github.com/Fullik")
+            }
+            developer {
+                id.set("maxsurovtsev")
+                name.set("Maxim Surovtsev")
+                url.set("https://github.com/maxsurovtsev")
+            }
             developer {
```

Comment:
But why is this the order again?) It's not random, not alphabetic, not just adding new members to the list, there is some logic in inserting

Reply:
:)